src/primevue/toast/toast.ts

```diff
@@ -0,0 +1,62 @@
+import { ToastPassThroughOptions } from "primevue/toast";
+import { tw } from "@/lib/tags.ts";
+
+const toast: ToastPassThroughOptions = {
+    message: ({ props }) => {
+        const base = tw`flex flex-row items-center gap-8 border-l-8 p-16`;
+
+        const severity = props.message?.severity;
+
+        const success = tw`border-l-green-900 bg-green-200`;
+        const error = tw`border-l-red-900 bg-red-200 text-red-900`;
+        const warn = tw`border-l-yellow-900 bg-yellow-200 text-yellow-900`;
+
+        return {
+            class: {
+                [base]: true,
+                [success]: severity === "success",
+                [error]: severity === "error",
+                [warn]: severity === "warn",
+            },
+        };
+    },
+
+    messageContent: () => ({
+        class: tw`flex flex-row items-start gap-8`,
+    }),
+
+    messageIcon: ({ props }) => {
```

Comment:
Todo: There is also a very strange behavior when a custom icon is there and I press "close": The icon switches between the custom and the default icon, and the toast does not disappear until I click several times:

https://github.com/user-attachments/assets/20286a99-9506-4ed5-abc4-7f16bd626d11

The border width seems to be changing, too.